packages/react/.storybook/preview.js

```diff
@@ -12,6 +12,7 @@ import {
   Stories,
   ArgTypes,
 } from '@storybook/addon-docs/blocks';
+import { allModes } from '../modes';
```

Comment:
```suggestion
import { allModes } from './modes';
```

is this path tripping up the CI?

Reply:
Seems like it, I've updated the correct path